src/app/tests/suites/TestCluster.yaml

```diff
@@ -2786,8 +2801,8 @@ tests:
       command: "readAttribute"
       attribute: "nullable_enum_attr"
       response:
-          saveAs: nullableEnumAttr254
-          value: 254
+          saveAs: nullableEnumAttr4
+          value: 4
```

Comment:
Why wuould this be 4?  It should be 3, just like the previous read, since the write did not happen.